tests/test_feature_extractor_static.py

```diff
@@ -9,13 +10,33 @@ def setUp(self):
         path_struct = "/Users/ehuber/Documents/ambiguous_alpaca/ambigous-alpaca/german-structgram-mincount-30-ctx-10-dims-300.fifu"
```

Comment:
no path hardcoding. we should put the embeddings in a directory in tests called "embeddings" then the test should take the relative path to the embeddings, stored in that directory.